tensorflow/compiler/xla/service/gpu/alpa_nccl_wrapper.cc

```diff
@@ -91,134 +101,175 @@ int SizeOfType(ncclDataType_t element_type) {
   }
 }
 
-std::shared_ptr< std::vector<ncclComm_t> > NcclInitCommunicator(std::vector<int> devices_vec) {
+tensorflow::StatusOr< std::shared_ptr< std::vector<ncclComm_t> > > NcclInitCommunicator(std::vector<int> devices_vec) {
+#if XLA_ENABLE_XCCL
     int n_devices = devices_vec.size();
     std::vector<ncclComm_t> comms;
     comms.resize(n_devices);
-    ncclCommInitAll(comms.data(), n_devices, devices_vec.data());
+    XLA_CUDA_RETURN_IF_ERROR(ncclCommInitAll(comms.data(), n_devices, devices_vec.data()));
     return std::make_shared< std::vector<ncclComm_t> >(std::move(comms));
+#else   // XLA_ENABLE_XCCL
+  return Unimplemented("NCCL support is not available.");
+#endif  // XLA_ENABLE_XCCL
 }
 
-void NcclLocalAllGather(std::vector<ncclComm_t> comms, 
-                        std::vector<PyBuffer::object> buffers, 
-                        std::vector<uint> local_start_positions, // TODO(hexu): is the range of uint too small?
-                        uint global_start, 
-                        uint n_elements) {
+tensorflow::Status NcclLocalAllGather(std::vector<ncclComm_t> comms, 
```

Comment:
It seems we don't need `tensorflow::`, we can just use `Status`

Reply:
OK